### ISSUE TITLE
[PNI] and [Foundation Site] Footer links

### ISSUE THREAD

**Issue by taisdesouzalessa** [mozilla/foundation.mozilla.org#4791]
### Background:
We will add a new link to the footer that will take users to the "Press Center" page - here is a [mockup of the page](https://foundation.mozilla.org/cms/pages/970/view_draft/) - it is in draft mode in production, make sure you are logged in Wagtail to see the page. With this change there is an opportunity to make all our footers consistent.

### Tasks:

- [ ] Add the "Press Center" link to both PNI and Foundation sites' footers.

- [ ] Reorder the items by alphabetical order (that works on Desktop, Tablet and Mobile views), except "Donate" label, which should be the first item due to its importance.

### Link:
The footer link will take people to this page: https://foundation.mozilla.org/cms/pages/970/view_draft/ (still in draft mode on production, make sure you are logged in Wagtail to see the page). The page will be published once we have all the approvals in place.

### Desired final state:
<img width="1363" alt="image" src="https://user-images.githubusercontent.com/22157921/84538347-5642fa00-aca6-11ea-99e3-4d376d981fe4.png">

@TheoChevalier tagging you here as this may have implications for localized pages. Kevin mentioned in another ticket that this page should be kept in English so people know that who will read it is an English speaker. However we may consider localize the label "Press Center" in all languages we have the footer on.

**Comment by taisdesouzalessa** [comment]
**Note:** let's wait on ticket #4868 to be resolved then open a new ticket with all footer changes at once.
